.vitepress/config.ts

```diff
@@ -65,6 +65,7 @@ export default withMermaid({
 
     search: {
       provider: 'local',
+      detailedView: true,
```

Comment:
ah sorry, it should be this:

```suggestion
      options: {
        detailedView: true
      }
```

Reply:
all good! thanks for maintaining vitepress, I'm really liking it so far